utils/config.py

```diff
@@ -41,107 +37,4 @@
 		"Fallen_Breath",
 		"Steve"
 	]
-}
```

Comment:
What's this deletion for

Reply:
i was having an issue with it giving me invalid username so i wasted a bunch of time going over random stuff before i realised i just needed to use email, anything not directly related is an artefact of me testing updated files from pycraft or something, anything not directly related to the packet stuff is just me being dumb